sentry/sentry.conf.example.py

```diff
@@ -363,3 +363,8 @@ def get_internal_network():
 # https://docs.djangoproject.com/en/4.2/ref/settings/#std-setting-CSRF_TRUSTED_ORIGINS
 
 # CSRF_TRUSTED_ORIGINS = ["https://example.com", "http://127.0.0.1:9000"]
+
+# If you would like to use self-hosted Sentry with only errors enabled, please set this
```

Comment:
Should we also define what's "errors only" and what's not? I mean it's clear you'd know what "errors only" means if you previously use Sentry. But I don't think newcomers will instantly understand about what features are available on the "feature complete" profile.

Reply:
Yep, I think we should. Going to also need to update docs, which I'll do today.

Reply:
See: https://develop.sentry.dev/self-hosted/#errors-only-self-hosted-sentry

Reply:
Thank you. I really need this feature!

Reply:
Yep, projects are always included.

Reply:
https://develop.sentry.dev/self-hosted/#errors-only-self-hosted-sentryDoes the "traces" mentioned in the document refer to "stack traces" and are they no longer supported under "errors only"?

Reply:
No, traces refers to the submenu inside the "Explore" menu. 
![image](https://github.com/user-attachments/assets/cb341378-154f-460c-a297-60eedb590763)

Stack traces is included on the errors-only since it's a crucial component to the error monitoring mechanism

Reply:
Thank you very much for your patient answers. I use Sentry as our team's crash analysis platform. It's really powerful and useful. The "errors only" version is just perfect for me.